[bitnami/elasticsearch] Elasticsearch enable security (#6010)
bitnami/charts: bitnami/elasticsearch/README.md

## Upgrading

### To 15.0.0

> This version or later are licensed under the Elastic License that is not currently accepted as a Open Source license by the Open Source Iniciative (OSI).

Reviewer: ```suggestion
From this version onwards, the chart is licensed under the Elastic License that is not currently accepted as an Open Source license by the Open Source Initiative (OSI).
```

> And include this version or later are including x-pack plugin installed by default.

Reviewer: ```suggestion
Also, from now on, the Helm Chart will include the x-pack plugin installed by default.
```
